Packs/ExtFilter/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### Test - ExtFilter Main
+- Modified task titles
```

Comment:
After moving the test playbook to the right directory, you can remove it from the changelog